stable/elasticsearch/README.md

```diff
@@ -64,7 +64,7 @@ The following table lists the configurable parameters of the elasticsearch chart
 | `appVersion`                         | Application Version (Elasticsearch)                                 | `6.4.2`                                             |
 | `image.repository`                   | Container image name                                                | `docker.elastic.co/elasticsearch/elasticsearch-oss` |
 | `image.tag`                          | Container image tag                                                 | `6.4.2`                                             |
-| `image.pullPolicy`                   | Container pull policy                                               | `IfNotPresent`                                      |
+| `image.pullPolicy`                   | Container pull policy                                               | `Always`                                            |
```

Comment:
Why the change to 'Always'?
This should only be used when we are following a latest tag which we are not.

Reply:
Actually this should reflect the default in values.

Reply:
The default is `IfNotPresent` so please revert this change.